.github/ISSUE_TEMPLATE/3-api-ref-docs-problem.md

```diff
@@ -0,0 +1,72 @@
+---
+name: "\U0001F4D7 Open an issue regarding the Node.js API reference docs"
+about: Let us know about any problematic API reference documents
+title: "doc: "
+labels: doc
+---
+
+# 📗 API Reference Docs Problem
+
+<!------------------------------------------------------------------------------
+Thank you for wanting to make nodejs.org better!
+
+This template is for issues with the Node.js API reference docs.
+
+For more general support, please open an issue in
+our help repo at “https://github.com/nodejs/help”.
+
+For the issue title, enter a one-line summary after “doc: ”.
+The “✍️” signifies a request for input. If unsure, do the best you can.
+
+If you found a problem with nodejs.org beyond the API reference docs, please
+open an issue in our website repo at “https://github.com/nodejs/nodejs.org”.
+------------------------------------------------------------------------------->
+
+<!--
+Version: output of “node -v”
+Platform: output of “uname -a” (UNIX), or version and 32 or 64-bit (Windows)
+Subsystem: if known, please specify affected core module name
+-->
+
+- **Version**: ✍️
+- **Platform**: ✍️
+- **Subsystem**: ✍️
+
+## Location
+
+_Section of the site where the content exists (or should)_
+
+Affected URL(s):
+- https://nodejs.org/api/✍️
+
+## Reproduction procedure
```

Comment:
If this is an issue with the reference docs, is this section applicable?

Reply:
So, currently, this issue template is geared more towards capturing reports about non-functional instructions, erroneous code samples, incorrectly documented APIs, and generally things that require one to execute a series of steps before discovering that there is indeed a documentation error. Currently, it would not be beneficial for capturing reports about grammatical mistakes, typos, poor wording, etc. As a compromise, do you think it would be appropriate to indicate that these sections are optional?